tgui/packages/tgui/interfaces/BlackMarketUplink.tsx

```diff
@@ -3,8 +3,39 @@ import { AnimatedNumber, Box, Button, Modal, Section, Stack, Tabs } from '../com
 import { formatMoney } from '../format';
 import { Window } from '../layouts';
 
+type BlackMarketUplinkContext = {
```

Comment:
there's only ever going to be one backend data sent here so it doesn't need to be wordier than "data". same for the other files

Reply:
Same, just use the name `Data` for all of them.